docs/build/program/apis/data-client.md

```diff
@@ -846,3 +846,232 @@ For more information, see the [Python SDK Docs](https://python.viam.dev/autoapi/
 
 {{% /tab %}}
 {{< /tabs >}}
+
+### CreateDataset
+
+Create a new dataset.
+
+{{< tabs >}}
+{{% tab name="Python" %}}
+
+**Parameters:**
+
+- name (str): The name of the dataset being created.
```

Comment:
```suggestion
- `name` [(str)](https://docs.python.org/3/library/stdtypes.html#text-sequence-type-str): The name of the dataset being created.
```
Typically we put the link on every type, and we use monospace for the parameter name